tests-behat/upload.feature

```diff
@@ -7,3 +7,13 @@ Feature: Upload
 
     When I select file input "file" with "Žlutý kůň" as "$kůň"
     Then Toast display should contain text "(name: $kůň, md5: b047fb155be776f5bbae061c7b08cdf0)"
+
+    When I click using selector "#atk_layout_maestro_form_form_layout_file_button"
```

Comment:
Two last things:
- `Context::elementAttributeShouldContainText` method seems to be introduced in quite illogical place in the file
- the ID selectors in Behat tests can change (for ex. if placed inside another layout, generator is changed, ..), please see other tests and use selectors based on `input name=file/img` element + inner button/img tag

all other changes are perfect and the coverage increase is perfect as well

Reply:
Really last ones?? :D

* Moved method to misc, not sure if that's better place to be
* Changed selectors, waiting on tests

Reply:
:D

the selectors still have the problem, the ID can be anything, base it on `<input name="img`, you can use xpath like `div[./input[@name='']]/...` to select owning div wit given `input` tag (see other Behat tests for examples)

Reply:
OMG I finally did that with selectors like `xpath(//div.action[.//input[@name='file']]//*[contains(concat(' ', normalize-space(@class), ' '), ' button ')][1])` :)